documentation/text-matching-ranking.html

```diff
@@ -207,6 +207,69 @@ <h3 id="prefix-search">Prefix Search</h3>
 
 
 
+<h2 id="match-configuration">Match Configuration</h2>
```

Comment:
As it stands, I think we should rename this section to something like Debugging Match Configuration. I think this kind of debugging is for an active developer, not the end user.